### DISCUSSION TITLE
More sources of reviews

### DISCUSSION THREAD

**Discussion by Root-FTW** [linaspurinis/mdblist.com#5] [Ideas]
**I found more sources of reviews that can help in MDBLIST.**

on this site i noticed they get reviews from major sites.
It would be very VERY great if mdblist would also consider those reviews.

[This is the site/example:](https://torrent-track.app/title/tt10365998)

[API example 🙂](https://api.torrent-track.app/meta/tt10365998.json)

[analyzing the json I think it uses the metacritic API to get those reviews (but I'm not 100% sure)](https://jsonhero.io/j/9EDX2dYai2YP)

**Comment by linaspurinis** [comment]
Thanx, it looks like easy-to-implement integration for additional reviews.

**Comment by linaspurinis** [comment]
It's not possible to mark the discussion as complete, so I will use the Issues section for new feature requests as well. 
Its closed: https://github.com/linaspurinis/mdblist.com/issues/6
